### PR TITLE
feat: Sql creation script

### PR DESCRIPTION
**Pull request by CjHare**
### Purpose for this PR
Initial create SQL for PostgreSQL.

### CODE REVIEW

File: scripts/sql/create.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+(
+    token_address     CHAR(20),
+    delegator_address CHAR(20),
+    trie_root         CHAR(32),
+    delegated_block   BIGINT,
+    PRIMARY KEY (token_address, delegator_address)
+);
+
+
+CREATE TABLE Token
```
Comment (CjHare) [inline scripts/sql/create.sql]: @joigno what is the `deployed_instance` being used for?

File: scripts/sql/create.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
+-- Delegatees are account that receive delegation of voting power from Delegators
+CREATE TABLE Delegation
+(
+    token_address     CHAR(20),
+    delegator_address CHAR(20),
+    delegatee_address CHAR(20),
+    proof             VARCHAR,
```
Comment (CjHare) [inline scripts/sql/create.sql]: @joigno question: what is the maximum size for the varchar? 

_This may be better as a text field_

**Comment by CjHare** [comment]
The intention is to get these in, then convert to use auto incrementing integers as primary key (also for foreign)

File: scripts/sql/create.sql (hunk 2 above)
Comment (jose-compu) [reply scripts/sql/create.sql]: there is no max size for the Proof. Maybe around 640 bytes for 1 million voters delegated for 1 single delegator.

File: scripts/sql/create.sql (hunk 1 above)
Comment (jose-compu) [reply scripts/sql/create.sql]: that is the implementation for upgradeable/proxied token

Comment (CjHare) [reply scripts/sql/create.sql]: why do you need the implementation contract address, isn't the interaction always done via the proxy?

_Doesn't interaction directly with the implementation behind a proxy defeat the purpose the token owner using the proxy?_

File: scripts/sql/create.sql (hunk 2 above)
Comment (CjHare) [reply scripts/sql/create.sql]: This is probably betters as a `TEXT` field then 🤔

File: scripts/sql/create.sql (hunk 1 above)
Comment (jose-compu) [reply scripts/sql/create.sql]: I think we can use only the token address and not the `deployed_instance`, maybe the deployed instance can be used in the Front-end for the users to check that they are voting on the current implementation of the Proposal.

File: scripts/sql/create.sql (hunk 2 above)
Comment (jose-compu) [reply scripts/sql/create.sql]: `TEXT` is fine for me.

**Review by jose-compu — APPROVED** [review]

File: scripts/sql/create.sql (hunk 1 above)
Comment (CjHare) [reply scripts/sql/create.sql]: > the deployed instance can be used in the Front-end for the users to check that they are voting on the current implementation of the Proposal.

That sound like you're suggesting its used to identify the proposal? entirely different to the token (Which is the name of the table).
Can we delete this table then?

**Pull request merged**